app/phone/src/main/java/dev/jdtech/jellyfin/PlayerActivity.kt

```diff
@@ -163,6 +174,13 @@ class PlayerActivity : BasePlayerActivity() {
                                 speedButton.imageAlpha = 255
                                 pipButton.isEnabled = true
                                 pipButton.imageAlpha = 255
+                                if (appPreferences.sleepMode) {
+                                    sleepModeButton.isEnabled = true
+                                    sleepModeButton.imageAlpha = 255
+                                } else {
+                                    sleepModeButton.isEnabled = false
+                                    sleepModeButton.imageAlpha = 0
+                                }
```

Comment:
Can do this instead here.

```suggestion
								sleepModeButton.isEnabled = appPreferences.sleepMode
                                sleepModeButton.imageAlpha = if (appPreferences.sleepMode) {
                                    255
                                } else {
                                    0
                                }
```